pkg/healthcheck/healthcheck.go

```diff
@@ -295,14 +295,14 @@ func (hc *HealthChecker) allCategories() []category {
 					description: "can create Roles",
 					hintURL:     "https://linkerd.io/2/getting-started/#step-0-setup",
 					check: func() error {
-						return hc.checkCanCreate("", "rbac.authorization.k8s.io", "v1beta1", "Role")
+						return hc.checkCanCreate(hc.ControlPlaneNamespace, "rbac.authorization.k8s.io", "v1beta1", "Role")
```

Comment:
gaah this was totally me in https://github.com/linkerd/linkerd2/pull/2078/files#diff-d4056ff163bcf2aeacefb2a34164563cR327 ... thanks for fixing !